libs/langchain/src/agents/middlewareAgent/types.ts

```diff
@@ -152,6 +152,11 @@ export interface ModelRequest {
    * The tools to make available for this step.
    */
   tools: (ServerTool | ClientTool)[];
+
+  /**
+   * Extra call options to pass to the model.
+   */
+  callOptions?: Record<string, unknown>;
```

Comment:
cc @christian-bromann  / @pokey is this a standard name in js?

@sydney-runkle this is extra config that we'll be passing to chat model invoke. We can either use the same standard name if it makes sense or follow whatever is convention in each language. 


* extra_params
* invoke_kwargs